operator-framework-junit5/src/main/java/io/javaoperatorsdk/operator/junit/OperatorExtension.java

```diff
@@ -135,6 +135,7 @@ protected void before(ExtensionContext context) {
     namespace += "-";
     namespace += context.getRequiredTestMethod().getName();
     namespace = KubernetesResourceUtil.sanitizeName(namespace).toLowerCase(Locale.US);
+    namespace = namespace.substring(0, Math.min(namespace.length(), 63));
```

Comment:
So the length is not taken into account in `sanitizeName`? That sounds like a fabric8 bug to me…

Reply:
Correct, what it does is:

```java
name.replaceAll("[^A-Za-z0-9]+", "-")
```

Reply:
Yep, just saw that, will fix it now :)